.github/workflows/zeebe-pr-benchmark.yaml

```diff
@@ -40,5 +41,7 @@ jobs:
           cluster_name: zeebe-cluster
           location: europe-west1-b
       - name: Delete benchmarks
+        env:
+          PR_HEAD_REF: ${{github.event.pull_request.head.ref}}
         run: |
-          kubectl delete ns ${{github.event.pull_request.head.ref}}-benchmark
+          kubectl delete ns "$PR_HEAD_REF"-benchmark
```

Comment:
:question: Out of curiosity, why is it better to do it this way?

Reply:
Github explains it on https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-an-intermediate-environment-variable (which I linked in the commit but it is not visible)